uportal-war/src/main/webapp/WEB-INF/jsp/Directory/directory.jsp

```diff
@@ -40,7 +40,11 @@
       <div class="portlet-section-body">
 
         <form action="${ formUrl }" method="POST">
-            <input name="query" value="${ fn:escapeXml(query )}"/> <input type="submit" value="<spring:message code='search.directory.submit'/>"/>
+            <label for="query">
+                 <spring:message code='search.directory'/>
+            </label>
+            <input name="query" value="${ fn:escapeXml(query )}"/>
```

Comment:
`<input>` tag needs and `id` attribute with value `query` for `<label>` tag to link correctly on screen readers.

``` html
<input id="query" name="query" value="${ fn:escapeXml(query )}"/>
```

ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for